src/lib/wallet.js

```diff
@@ -16,26 +15,21 @@ import Logger from './logger';
 
 async function initMiddleware() {
   const { middlewareUrl } = store.getters.activeNetwork;
-  const swag = await fetchJson(`${middlewareUrl}/middleware/api`);
-  swag.paths['/names/auctions/{name}/info'] = {
-    get: {
-      operationId: 'getAuctionInfoByName',
-      parameters: [
-        {
-          in: 'path',
-          name: 'name',
-          required: true,
-          type: 'string',
-        },
-      ],
-    },
+
+  const fetchMiddleware = async (url, field) => (await fetchJson(url))[field];
+  const middleware = {
```

Comment:
Doesn't the new middleware have a swagger file? The convention of field names case was made by Swagger stamp and I would rather write the swagger file by myself than to rewrite all to snake_kase.

Reply:
They has actually https://mainnet.aeternity.io/mdw/swagger/swagger.json, I'll rewrite this part